core/server/src/main/java/alluxio/master/file/FileSystemMaster.java

```diff
@@ -2473,21 +2473,25 @@ public FileSystemCommand workerHeartbeat(long workerId, List<Long> persistedFile
     }
     // If the file is persisted in UFS, also update corresponding owner/group/permission.
     if ((ownerGroupChanged || permissionChanged) && inode.isPersisted()) {
-      MountTable.Resolution resolution = mMountTable.resolve(inodePath.getUri());
-      String ufsUri = resolution.getUri().toString();
-      UnderFileSystem ufs = resolution.getUfs();
-      if (ownerGroupChanged) {
-        try {
-          ufs.setOwner(ufsUri, inode.getOwner(), inode.getGroup());
-        } catch (IOException e) {
-          throw new AccessControlException("Could not setOwner for UFS file " + ufsUri, e);
+      if ((inode instanceof InodeFile) && !((InodeFile) inode).isCompleted()) {
+        LOG.debug("Alluxio does not propagate chown/chgrp/chmod to UFS for incomplete files.");
```

Comment:
We need to support permission modification in Alluxio space because Hadoop `create` file with permission is [implemented] (https://github.com/apache/hadoop/blob/trunk/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/FileSystem.java#L591) as follows: 

```
  public static FSDataOutputStream create(FileSystem fs,
      Path file, FsPermission permission) throws IOException {
    // create the file with default permission
    FSDataOutputStream out = fs.create(file);
    // set its permission to the supplied one
    fs.setPermission(file, permission);
    return out;
  }
```